spec/tests/ColumnLineageDatasetFacet/1.json

```diff
@@ -1,30 +1,47 @@
 {
   "columnLineage": {
     "fields": {
-      "order_day_of_week": {
+      "NAME": {
         "inputFields": [
           {
-            "field": "order_placed_on",
-            "name": "public.top_delivery_times",
-            "namespace": "postgres://postgres:5432"
-          }
-        ],
-        "transformationDescription": "",
-        "transformationType": ""
-      },
-      "order_placed_on": {
-        "inputFields": [
+            "namespace": "SnowflakeOpenLineage",
+            "name": "CUSTOMERS",
+            "field": "NAME",
+            "transformations": [
+              {
+                "type": "DIRECT",
```

Comment:
could you add subtype `IDENTITY` here? assuming that you want to stick to the example you gave in the PR description

Reply:
Added! (not sure if I should resolve this convo or if you should after re-review?)